aopy/analysis.py

```diff
@@ -111,9 +113,18 @@ def get_preferred_direction(b1, b2):
     .. math:: 
         
         arctan(\\frac{b_1^2}{b_2^2})
-        
+    
+    Returns:
+        float: Preferred direction of the curve fit in radians
```

Comment:
This function is not returning preferred direction in radians anymore. Any reason to covert pd to degrees here instead of converting outside the function?